src/vm/docs/mod.rs

```diff
@@ -91,6 +91,27 @@ const NONE_KEYWORD: KeywordAPI = KeywordAPI {
 "
 };
 
+const TRUE_KEYWORD: KeywordAPI = KeywordAPI {
+    name: "true",
+    output_type: "(optional ?)",
```

Comment:
Output type should be `bool`

Reply:
Good catch. Thanks. Resolved in 46f70fc.

Perhaps inconsistencies in documented output_type could be caught in unit testing.